contrib/document_cleanup/light_weight_document_cleanup_ICDAR2021/README.md

```diff
@@ -0,0 +1,25 @@
+# Document Image Cleanup
```

Comment:
Fyi, this might benefit from a "how to get started" section (e.g. answer: run the jupyter notebook) and maybe a screenshot or two.